scripts/obtain_user_auth.py

```diff
@@ -1,65 +0,0 @@
-# Copyright 2016 Google Inc.
```

Comment:
Is it intended to delete this file? How are we system testing user-based credentials files?

Reply:
The removal of `scripts/obtain_user_auth.py` was intentional. The [current script](https://github.com/googleapis/google-auth-library-python/blob/master/scripts/obtain_user_auth.py) uses oauth2client to make the credentials. I added instructions on how to get the credentials using gcloud in the 'Running system tests' section of the `CONTRIBUTING.rst`.

https://github.com/googleapis/google-auth-library-python/blob/use-new-nox/CONTRIBUTING.rst#running-system-tests